package.json

```
@@ -20,19 +20,19 @@
     ]
   },
   "dependencies": {
-    "@hapi/boom": "9.x.x",
-    "@hapi/bourne": "2.x.x",
-    "@hapi/content": "^5.0.2",
-    "@hapi/file": "2.x.x",
-    "@hapi/hoek": "9.x.x",
-    "@hapi/pez": "^5.0.1",
-    "@hapi/wreck": "17.x.x"
+    "@hapi/boom": "^10.0.0",
+    "@hapi/bourne": "^3.0.0",
+    "@hapi/content": "^6.0.0",
+    "@hapi/file": "^2.0.0",
```

Pending https://github.com/hapijs/file/pull/14
```suggestion
    "@hapi/file": "^3.0.0",
```